core/controllers/contributor_dashboard.py

```diff
@@ -307,14 +307,17 @@ def _get_reviewable_exploration_opportunity_summaries(
             for topic in topics
             for reference in topic.get_all_story_references()
             if reference.story_is_published])
-        in_review_suggestions = (
-            suggestion_services.get_reviewable_translation_suggestions(user_id))
-        in_review_suggestion_target_ids = [
+        in_review_suggestions, _ = (
+            suggestion_services
+            .get_reviewable_translation_suggestions_by_offset(
+                user_id, None, None, 0))
+        # Filter out suggestions that should not be shown to the user.
+        in_review_suggestion_target_ids = {
             suggestion.target_id
             for suggestion in
             suggestion_services.get_suggestions_with_translatable_explorations(
                 in_review_suggestions)
-        ]
+        }
```

Comment:
Maybe add a comment explaining why you're using a set here (is there a possibility of duplication)?

Reply:
Added a comment. The set isn't strictly necessary, but we only care about the unique IDs and there can be a lot of duplicates.